pyadjoint/reduced_functional.py

```diff
@@ -4,6 +4,36 @@
 from .overloaded_type import OverloadedType
 
 
+def get_extract_derivative_components(derivative_components):
```

Comment:
```suggestion
def _get_extract_derivative_components(derivative_components):
```

Please make these names private.

Reply:
fixed in latest commit.